gcp_variant_transforms/libs/vcf_field_conflict_resolver_test.py

```diff
@@ -34,40 +37,164 @@ def _field_count(self, symbol):
+         'field_data': ['1', '2'], 'resolved_field_data': [True, True]},
+        {'schema_type': TableFieldConstants.TYPE_INTEGER,
+         'schema_mode': TableFieldConstants.MODE_NULLABLE,
+         'field_data': '1', 'resolved_field_data': 1},
+        {'schema_type': TableFieldConstants.TYPE_INTEGER,
+         'schema_mode': TableFieldConstants.MODE_REPEATED,
+         'field_data': ['1', '2'], 'resolved_field_data': [1, 2]},
+        {'schema_type': TableFieldConstants.TYPE_FLOAT,
+         'schema_mode': TableFieldConstants.MODE_NULLABLE,
+         'field_data': 1, 'resolved_field_data': float(1)},
+        {'schema_type': TableFieldConstants.TYPE_FLOAT,
+         'schema_mode': TableFieldConstants.MODE_REPEATED,
+         'field_data': [1, 2], 'resolved_field_data': [float(1), float(2)]},
+        {'schema_type': TableFieldConstants.TYPE_STRING,
+         'schema_mode': TableFieldConstants.MODE_NULLABLE,
+         'field_data': 1, 'resolved_field_data': '1'},
+        {'schema_type': TableFieldConstants.TYPE_STRING,
+         'schema_mode': TableFieldConstants.MODE_REPEATED,
+         'field_data': [1, 2], 'resolved_field_data': ['1', '2']},
+        {'schema_type': TableFieldConstants.TYPE_BOOLEAN,
+         'schema_mode': TableFieldConstants.MODE_NULLABLE,
+         'field_data': '', 'resolved_field_data': False},
+        {'schema_type': TableFieldConstants.TYPE_BOOLEAN,
+         'schema_mode': TableFieldConstants.MODE_REPEATED,
+         'field_data': ['', ''], 'resolved_field_data': [False, False]},
+        {'schema_type': TableFieldConstants.TYPE_STRING,
+         'schema_mode': TableFieldConstants.MODE_NULLABLE,
+         'field_data': [], 'resolved_field_data': None},
+        {'schema_type': TableFieldConstants.TYPE_STRING,
+         'schema_mode': TableFieldConstants.MODE_REPEATED,
+         'field_data': [], 'resolved_field_data': []},
+        {'schema_type': TableFieldConstants.TYPE_BOOLEAN,
+         'schema_mode': TableFieldConstants.MODE_NULLABLE,
+         'field_data': [], 'resolved_field_data': False},
+        {'schema_type': TableFieldConstants.TYPE_BOOLEAN,
+         'schema_mode': TableFieldConstants.MODE_REPEATED,
+         'field_data': [], 'resolved_field_data': []},
+    ]
+
+    for config in test_data_configs:
+      self.assertEqual(
+          self._resolver.resolve_schema_conflict(
+              bigquery_schema_descriptor.FieldDescriptor(
+                  type=config['schema_type'],
+                  mode=config['schema_mode']),
+              config['field_data']),
+          config['resolved_field_data'])
+
+    with self.assertRaises(ValueError):
+      self._resolver.resolve_schema_conflict(
+          bigquery_schema_descriptor.FieldDescriptor(
+              type=TableFieldConstants.TYPE_INTEGER,
+              mode=TableFieldConstants.MODE_NULLABLE),
+          'foo')
+      self.fail('Should raise exception for converting str to int')
+
+  def test_resolving_schema_conflict_number(self):
+
```

Comment:
nit: please remove the extra blank line after def.

Reply:
Done!